src/appmixer/mailerlite/core/FindSubscribers/component.json

```
@@ -0,0 +1,75 @@
+{
+    "name": "appmixer.mailerlite.core.FindSubscribers",
```

```suggestion
    "name": "appmixer.mailerlite.core.ListSubscribers",
```
This one should be called List because there is no Query or any other filter that would change the output data amount.

---

I have added some filters so this is now findSubscribers in true sense